src/modules/project/pages1/projectView/views/body/sections/header/components/LightningAddressModal.tsx

```diff
@@ -0,0 +1,84 @@
+import { Button, ButtonProps, ListItem, UnorderedList, VStack } from '@chakra-ui/react'
+import { useTranslation } from 'react-i18next'
+import { PiLightning } from 'react-icons/pi'
+
+import { Modal } from '@/shared/components/layouts'
+import { useModal } from '@/shared/hooks'
+
+import { Body } from '../../../../../../../../../shared/components/typography'
+import { LightningAddress } from './LightningAddress'
+import { NpubDisplay } from './NpubDisplay'
+
+interface LightningAddressModalProps extends ButtonProps {
+  name: string
+  npub?: string
+}
+
+export const GEYSER_DOMAIN_POSTFIX = '@geyser.fund'
```

Comment:
Same here, I would make an env variable of it

Reply:
Only one would be needed, I will move it to constants folder. Like I mentioned this does not need to be an env variable.